tests/e2e/tests/content-manager/preview.spec.ts

```diff
@@ -93,7 +94,7 @@ describeOnCondition(edition === 'EE')('Preview', () => {
     await clickAndWait(page, page.getByRole('link', { name: /open preview/i }));
 
     // Check if the iframe is present
-    const iframe = await page.getByTitle('Preview');
+    const iframe = page.getByTitle('Preview');
     expect(iframe).not.toBeNull();
```

Comment:
Shouldn't you keep the await or await the expect? Per the example: https://playwright.dev/docs/api/class-page#page-get-by-title